src/components/sections/Header.tsx

```diff
@@ -0,0 +1,12 @@
+interface HeaderProps {
+    className: string;
+    children: string;
+}
+
+const Header: React.FC<HeaderProps> = ({className, children}) => {
```

Comment:
Why do you expect the header to take children?

Reply:
I don't, just rendered a chilren to see something on my localhost.